style.css

```diff
@@ -651,3 +597,119 @@ textarea {
   display: flex;
   margin-top: 25px;
 }
+
+nav {
+  color: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav .mainMenu {
+  display: flex;
+  list-style: none;
+  margin-top: -8px;
+}
+
+nav .mainMenu li a {
+  font-family: "Roboto", Arial, sans-serif;
+  display: inline-block;
+  padding: 15px;
+  text-decoration: none;
+  color: #fff;
+  font-size: 16px;
+}
+
+nav .openMenu {
+  font-size: 2rem;
+  margin: 20px;
+  display: none;
+  cursor: pointer;
+}
+
+nav .mainMenu .closeMenu,
+i {
+  font-size: 25px;
+  display: none;
+  cursor: pointer;
+}
+
+nav .logo {
+  margin: 6px;
+  font-size: 18px;
+  cursor: pointer;
+}
+
+.fa-times:hover {
+  color: #fff;
+}
+
+.fa-bars:hover {
+  color: #fff;
+}
+
+@media (max-width: 767px) {
+  nav .mainMenu {
+    height: 101.5vh;
+    position: fixed;
+    font-size: 55px;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    z-index: 10;
+    flex-direction: column;
+    justify-content: center;
+    gap: 15px;
+    background-color: #191765;
+    transition: top 1s ease;
+    background-image: url('images/shapes-mobile-menu.svg');
+    background-repeat: no-repeat;
+    background-size: 100%;
+    display: none;
+  }
+
+  #envelope {
+    display: none;
+  }
+
+  .navLink {
+    font-size: 40px !important;
+  }
+
+  .navLink::after {
+    content: url("images/right.svg");
+    width: 12px;
+    height: 12px;
+    fill: #fff;
+    position: relative;
+    right: 0;
+  }
+
+  nav .mainMenu li a {
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 24px;
+    line-height: 32px;
+    line-height: 114%;
+    text-align: left;
+    display: flex;
+    justify-content: space-between;
+    margin-right: 30px;
+  }
+
+  nav .mainMenu .closeMenu {
+    display: block;
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    margin-top: 25px;
+  }
```

Comment:
- [ ] Please use the correct size for your close(X) button, and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/161624896-5adeccda-4e73-47a9-9e7f-0b124a841026.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/161624877-5073f0d0-0052-4b17-b0e6-29f65b3c6c48.png)